js/ez-platform-ui-app.js

```diff
@@ -152,12 +153,21 @@
         }
 
         /**
-         * Creates ez-universal-discovery custom element and sets properties and handlers
+         * Adds an event listener for `ez:navigateTo` event with the url of the location where the app should navigate to.
```

Comment:
should be something like:

*Adds an event listener for the `ez:navigateTo` event to navigate to the url provided by the 
event details*